coc/enums.py

```diff
@@ -262,7 +264,7 @@ def in_game_name(self) -> str:
 
 SPELL_ORDER = ELIXIR_SPELL_ORDER + DARK_ELIXIR_SPELL_ORDER
 
-HOME_BASE_HERO_ORDER = ["Barbarian King", "Archer Queen", "Grand Warden", "Royal Champion"]
+HOME_BASE_HERO_ORDER = ["Barbarian King", "Archer Queen", "Grand Warden", "Royal Champion", "Minion Prince"]
```

Comment:
I would vote to keep it the order of unlocking, matching the ingame order in the hero hut, so minion prince in the middle